tendrl/gluster_integration/message/callback.py

```diff
@@ -0,0 +1,12 @@
+class Callback(object):
+    def peer_connect(self, event):
```

Comment:
have verified that when a "PEER_CONNECT" event comes to gluster-integration the control comes here, we have to add code to perform some action when we receive this event

Reply:
This can be linked with https://github.com/Tendrl/commons/pull/711 to invoke flow to update dashboard. Events are the right place to consume the utility.